docs/modules/data-structures/pages/distributed-data-structures.adoc

```diff
@@ -211,6 +211,10 @@ Hazelcast is designed to create any distributed data structure whenever it is ac
 Therefore, keep in mind that a
 data structure is recreated when you perform an operation on it even after you have destroyed it.
 
+NOTE: If you plan to use the distributed object later, prefer clearing the object using `clear()` method or equivalent instead of destroying it.
```

Comment:
```suggestion
NOTE: If you plan to use the distributed object later, clear the object using the `clear()` method or equivalent instead of destroying it.
```